From 28922d00f3b9271fd8a234a517598c2220b393dd Mon Sep 17 00:00:00 2001
From: Filip Pytloun <filip@pytloun.cz>
Date: Mon, 27 Jun 2016 15:35:48 +0200
Subject: Use libdatrie-dev for build

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index eea48b6..cf002a0 100755
--- a/setup.py
+++ b/setup.py
@@ -6,7 +6,7 @@ import os
 
 from setuptools import setup, Extension
 
-LIBDATRIE_DIR = 'libdatrie'
+LIBDATRIE_DIR = '/usr/include/datrie'
 LIBDATRIE_FILES = glob.glob(os.path.join(LIBDATRIE_DIR, "datrie", "*.c"))
 
 DESCRIPTION = __doc__
